WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/breakpoint/BreakpointChangeType.java

```diff
@@ -0,0 +1,46 @@
+package nl.tudelft.watchdog.core.logic.breakpoint;
+
+import com.google.gson.annotations.SerializedName;
+
+/** The different types of changes to a breakpoint. */
+public enum BreakpointChangeType {
+	/** Breakpoint has been enabled. */
+	@SerializedName("en")
+	ENABLED,
+
+	/** Breakpoint has been disabled. */
+	@SerializedName("ds")
+	DISABLED,
+	
+	/** Hit count added on breakpoint. */
```

Comment:
A hit count doesn't specify how often a bp is hit, but at what hit the program should actually be suspended. From Eclipse's doc: "If a breakpoint has a hit count of N, execution will suspend when the breakpoint is encountered for the Nth time. After being hit, the breakpoint is disabled until either it is re-enabled or its hit count is changed." (http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.jdt.doc.user%2Freference%2Fbreakpoints%2Fref-hitcount_option.htm)

So, with our analysis when can identify how often this feature is actually used.